Lib/tkinter/__init__.py

```diff
@@ -764,14 +764,32 @@ def after_cancel(self, id):
         Identifier returned by after or after_idle must be
         given as first parameter."""
         try:
-            data = self.tk.call('after', 'info', id)
+            data = self.after_info(id)
             # In Tk 8.3, splitlist returns: (script, type)
             # In Tk 8.4, splitlist may return (script, type) or (script,)
```

Comment:
Check that these comments still are related.

Reply:
I couldn't find any Tk/Tcl documentation that referenced this.  Here is the commit that added the comment:
https://github.com/python/cpython/commit/3c0f2c91adf086f809169f94ca9a2a75df3dc996

Maybe `after_cancel` had been called with `None`, which wouldn't have returned the 2-tuple?